servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ConnectionContext.java

```diff
@@ -24,7 +24,7 @@
 import javax.net.ssl.SSLSession;
 
 /**
- * A service execution context.
+ * A context that represents a network connection.
```

Comment:
`that represents a network connection.` => `for a connection`

As the context may not represent the entire connection.